app/views/recipes/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="container">
+  <h1>Create your recipe</h1>
+
+  <%= form_with(model: @recipe, url:user_recipes_path(@user), local: true, method: :post) do |form| %>
+    <div>
+      <%= form.label :name %><br>
+      <%= form.text_field :name, autocomplete: 'off' %>
+    </div>
+
+    <div>
+      <%= form.label :description %><br>
+      <%= form.text_area :description %>
+    </div>
+
+    <div>
+      <%= form.label :preparation_time %><br>
+      <%= form.number_field :preparation_time, in: 1..1000, step: 1, autocomplete: "off" %>
+    </div>
+
+    <div>
+      <%= form.label :cooking_time %><br>
+      <%= form.number_field :cooking_time, in: 1..1000, step: 1, autocomplete: "off" %>
+    </div>
+
+    <div>
+      <%= form.label :public %><br>
+      <%= form.check_box :public %>
+    </div>
+
```

Comment:
I noticed you added validation to all your forms, which is a good practice😉. The issue currently is that, for all your input fields or forms, whenever a user proceeds to submit with any empty field, it breaks the app. This can lead to a poor user experience. If a field is not provided, I suggest you send an error message to the user on the same page or you can use the `required` HTML attribute that prevents the user from submitting the form automatically😉.

In summary, let's find a way of preventing the user from submitting a form with empty fields, but without breaking the app👍🏼. Please remember to implement this for all your forms😉.

<img width="1125" alt="image" src="https://github.com/tsheporamantso/Recipe-App/assets/64643995/7a3d815d-46a5-401f-8ad9-52a53fb31618">